oeps/processes/oep-0055/decisions/0001-use-backstage-to-support-maintainers.rst

```diff
@@ -0,0 +1,126 @@
+0001 Use Backstage to Support Maintainers
+#########################################
+
+Status
+******
+
+**Draft**
+
+.. TODO Upon Acceptance
+
+	* Update OEP to mention the catalog-info.ymal file instead of the readme.
```

Comment:
did you mean yaml?
```suggestion
	* Update OEP to mention the catalog-info.yaml file instead of the readme.
```

Reply:
Yep, definitely did. This will resolve itself because this comment is only here as a reminder to me to update the OEP and will be removed once we've aligned on the rest of the content of this file.